picodom.d.ts

```diff
@@ -0,0 +1,30 @@
+declare namespace picodom {
+  export interface VNode<D> {
+    tag: string
+    data?: D
+    children: VNode<{}>
+  }
+
+  export interface Component<D> {
+    (data?: D, ...children: VNode<{}>[]): VNode<D>
+  }
+
+  export type VElement<D> = Component<D> | string
```

Comment:
Yes but why? It's been helpful with the React type definitions to have this type exist. I actually think it needs to be applied to more places.